scripts/generate/update-package-json.ts

```diff
@@ -0,0 +1,43 @@
+import fs from 'fs';
+import path from 'path';
+import {promisify} from 'util';
+import {Template} from './generate-src';
+
+export function addScripts({entry, type}: {entry: string, type: string}) {
+  return updatePackage((npmPackage) => {
+    npmPackage.scripts['server'] = `argo-admin-cli server --entry="${entry}" --port=39351 --type=${type}`;
```

Comment:
Can we update the README to mention how these 2 scripts were added and how they can configure it. We can link to the `argo-admin-cli` docs: https://www.npmjs.com/package/@shopify/argo-admin-cli

Reply:
Wow these docs are already out of date. I gotta update the argo-admin-cli because it no longer uses `init`. Not to block this PR, I'm going to point to the npm docs anyway, and I'll do an upcoming README change.

Reply:
https://github.com/Shopify/argo-admin-cli/pull/2